solrman/smmodel/model.go

```diff
@@ -285,8 +288,7 @@ func (m *Model) computeNextMove(immobileCores []bool) *Move {
 				// no good choices
 				break
 			}
-			if target.MaxSize > 0 && target.Size >= target.MaxSize {
-				// too much data already on this node
+			if core.Size+target.Size > target.MaxSize {
```

Comment:
We are passing 90% of disk size, thus just make after move it should cross that size

Reply:
Nit: This should probably retain the check for `target.MaxSize > 0` as well.